### PR TITLE
Add new iidm hvdc extensions

### PR DESCRIPTION
**Pull request by pl-buiquang**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
feature

**What is the new behavior (if this is a feature change)?**
Add new iidm hvdc related extensions : HvdcOperatorActivePowerRange and HvdcAngleDroopPowerControl.

### CODE REVIEW

**Review by MioRtia — COMMENTED** [review]
It is okay for me but just to be sure: when the author was someone else, was it because you moved it here? Maybe add your name as well as an author too.

**Comment by pl-buiquang** [comment]
Yes original author was someone else. Theses classes are from internal repositories.

File: iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/NetworkXml.java [hunk 1]
```diff
@@ -38,6 +38,7 @@
 import javax.xml.validation.SchemaFactory;
 import javax.xml.validation.Validator;
 import java.io.*;
+import java.nio.charset.Charset;
```
Comment (MioRtia) [inline iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/NetworkXml.java:41]: Please delete this unused import

**Review by MioRtia — CHANGES_REQUESTED** [review]
I think there was a small mistake in the commit :) But otherwise, it is okay for me, I am merging this once this is corrected

**Review by MioRtia — APPROVED** [review]

**Pull request merged**